c2corg_ui/static/partials/cards/articles.html

```diff
@@ -1,36 +1,59 @@
-    </div>
-
-    <b class="article-type margin-t-sm margin-b-sm"><span class="green-text glyphicon glyphicon-tag margin-r-sm"></span>{{doc['article_type'] | translate}}</b>
-
-    <div class="article-categories">
-      <b ng-repeat="category in ::doc.categories" class="value">{{category | translate }}{{$last ? '' : ', '}}</b>
-    </div>
-
-    <div class="activities flex wrap-row margin-t-sm">
-      <span ng-repeat="activity in ::doc.activities | limitTo: 4"  class="activity icon-{{::activity}}"
-            uib-tooltip="{{cardCtrl.translate(activity)}}" tooltip-append-to-body="true"></span>
-
-      <span class="flex flexend-align show-more-activities"
-            ng-if="doc.activities.length > 4 && !cardCtrl.remainingActivities"
-            ng-click="$event.preventDefault(); cardCtrl.remainingActivities = true">
-        ...(+ {{::doc.activities.length - 4}})
-      </span>
-
-      <span ng-if="cardCtrl.remainingActivities" ng-repeat="activity in ::doc.activities | limitTo: (4 - doc.activities.length)"
-            class="activity icon-{{::activity}}" uib-tooltip="{{cardCtrl.translate(activity)}}" tooltip-append-to-body="true">
-      </span>
+<article class="post card-article" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;" ng-click="cardCtrl.openDoc(doc)">
+  <a ng-href="{{::cardCtrl.createURL(doc)}}" title="{{::feedCtrl.showTitle(doc['document']['locales'])}}" ng-click="$event.stopPropagation()" class="text-link"> 
+    <div class="parent-text">
+      <div class="text emptyimg" ng-if="::!doc.image1">
+        <div class="title">{{::locale.title}}</div>
+        <div class="summary"><span ng-repeat="category in ::doc.categories" class="value">{{category | translate }}{{$last ? '' : ', '}}</span></div>
+      </div>
+    </div></a><div class="parent-img"><div class="img" ng-if="::doc.image1"><a ng-href="/images/{{::doc.image1.document_id}}/{{::doc.image1.locales[0].lang}}"><img class="imgo" ng-src="{{feedCtrl.createImageUrl(doc.image1.filename, 'MI')}}" alt="Photo"></a></div></div><div class="parent-infop">
+  </div>
+  <div class="parent-infop parent-infop-first" ng-if="::doc['article_type'] != null">
+    <div class="infop-bar">
+      <div class="parent-bubble">
+        <div class="bubble">
+          <span class="glyphicon glyphicon-tag"></span>
+        </div>
+      </div>
+      <div class="text-bubble">
+        {{doc['article_type'] | translate}}
+      </div>
+      <div class="grow">
+      </div>
     </div>
-
-    <div class="quality" ng-if="::doc['quality']"
-         uib-tooltip="{{'quality'| translate}} : {{doc['quality'] | translate}}" tooltip-append-to-body="true">
-      <span class="quality-{{::doc['quality']}}"></span>
+  </div>
+  <div class="parent-infop">
+    <div class="infop-bar">
+      <div class="parent-activity">
+        <div class="parent-bubble" ng-repeat="activity in ::doc.activities" >
+          <a href="#" class="shcom" uib-tooltip="{{activity | translate}}"> 
```

Comment:
Why do you need all those ``href="#"``? Why do you need links? (ditto in the other partials)